include/glow/Runtime/RuntimeTypes.h

```diff
@@ -171,6 +171,30 @@ struct HostConfig {
   size_t executorThreads{3};
 };
 
+/// This is struct for user defined partition.
+struct PartitionConfig {
+  /// The name of the function to be partitioned.
+  std::string funcName;
+  /// The number of user defined partitions.
+  /// The partition ids are between 0 and numOfPartitions - 1, inclusive.
+  size_t numOfPartitions;
+  /// The backend for each partition. backendNames.size() == numOfPartitions.
```

Comment:
This means backendNames.size() should be equal to numOfPartitions. Otherwise, there is an error.

Reply:
Can't we just have backendNames alone and the size tells how many partitions.

Reply:
It also works. The reason I added this separate field is: I think it is clear for people just use a number directly as a reference. The we compare the size of partition names and backends in case something is missing by accident.